addons/im_livechat/tests/test_chatbot_form_ui.py

```diff
@@ -5,6 +5,10 @@
 from odoo.addons.base.tests.common import HttpCaseWithUserDemo
 
 
+def step_text_to_markup_text(step_text):
+    return f'<p>{step_text}</p>'
```

Comment:
Not really useful, indirection makes reading the test more confusing, better have `p` directly in the test itself, especially when the method name is longer than the text it replaces.